INCHI-1-SRC/INCHI_BASE/src/strutil.c

```diff
@@ -4019,43 +4019,44 @@ int remove_terminal_HDT( int num_atoms, inp_ATOM *at, int bFixTermHChrg )
 }
 
 
-
-/****************************************************************************/
-int get_iat_number( int el_number, const int el_num[], int el_num_len )
-{
-    int i;
-    for (i = 0; i < el_num_len; i++)
-    {
-        if (el_num[i] == el_number)
-        {
-            return i;
-        }
-    }
-
-    return -1;
-}
-
-
 /*#endif*/ /* } DISCONNECT_SALTS */
 
 typedef enum tagIonAtomType
 {
     IAT_H = 0,
-    IAT_C,
-    IAT_N,
-    IAT_P,
-    IAT_O,
-    IAT_S,
-    IAT_Se,
-    IAT_Te,
-    IAT_F,
-    IAT_Cl,
-    IAT_Br,
+    IAT_C,   
```

Comment:
Minor nit: trailing whitespace.

Reply:
Fixed

Reply:
Hi @johnmay,
Please do not worry or waste your time on correcting whitespaces as the original code is crammed with them as well as chaotic new lines, tabs, etc. I have not even bothered to look into this.
I guess that the display resolution at the time was low.